src/styles/global.css

```diff
@@ -58,3 +58,11 @@ h6 {
 html {
 	@apply bg-slate-200;
 }
+
+.fit-height {
+	height: 100%;
+}
+
+.fit-width {
+	width: 100%;
+}
```

Comment:
```suggestion
```
We should avoid writing custom CSS where ever possible. Makes it easier to maintain the code :)